kcidb/db/__init__.py

```diff
@@ -441,17 +441,23 @@ class ArgumentParser(kcidb.misc.ArgumentParser):
     Command-line argument parser with common database arguments added.
     """
 
-    def __init__(self, *args, database=None, **kwargs):
+    def __init__(self, driver_types, *args, database=None, **kwargs):
```

Comment:
GitHub still doesn't allow commenting on unchanged lines in PRs, but this comment belongs to line 407:

This commit (and not the next one as it is right now) needs to also make DBHelpAction actually use the `driver_types` from the parsers (instead of using the global DRIVER_TYPES). Otherwise this change is incomplete, and the classes are not ready for extraction into their own `argparse` module.